src/components/app-top-bar.js

```diff
@@ -153,6 +153,8 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
 
     const workingNode = useSelector((state) => state.workingTreeNode);
```

Comment:
Strange to keep workingNode here then.
It means that we show the selectedNode in topBar but we use the workingNode for the search feature... it's a mess.